library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -28,6 +30,7 @@ internal constructor(
         private var endpointUrl = DatadogEndpoint.SESSION_REPLAY_US1
         private var privacy = SessionReplayPrivacy.MASK_ALL
         private var extensionSupport: ExtensionSupport = NoOpExtensionSupport()
+        private var samplingRate: Float = 0f
```

Comment:
for the naming consistency

```suggestion
        private var sampleRate: Float = 0f
```